Add XMPP WebSocket / Stream Management support (#502)
jitsi/docker-jitsi-meet: README.md

`JIGASI_TRANSCRIBER_ADVERTISE_URL` | Jigasi will post an URL to the chat with transcription file | true
`DISABLE_HTTPS` | Handle TLS connections outside of this setup | 1
`ENABLE_HTTP_REDIRECT` | Redirect HTTP traffic to HTTPS (necessary for Let's Encrypt) | 1

> `ENABLE_WEBSOCKET` | Enabled xmpp traffic over websockets | 0

Reviewer: Leet's call this ENABLE_XMPP_WEBSOCKET, since JVB datachannels can also be configured to use WS.

Author: Done

The public IP address is discovered via [STUN]. STUN servers can be specified with the ``JVB_STUN_SERVERS``
option.

### Prosody WebSocket Support

By default communication with Prosody is done using HTTP.

If `ENABLE_XMPP_WEBSOCKET` is set to `1` the preferred communication method is WebSocket. HTTP is still available as a fallback.

If Jitsi runs behind a reverse proxy make sure the reverse proxy configuration supports WebSocket.

Read more: http://nginx.org/en/docs/http/websocket.html

> 

Reviewer 2: These changes need to be applied to https://github.com/jitsi/handbook/blob/master/docs/devops-guide/docker.md (cf: #601)

The public IP address is discovered via [STUN]. STUN servers can be specified with the ``JVB_STUN_SERVERS``
option.

> ### Prosody Websocket Support

Reviewer: ```suggestion
### Prosody WebSocket Support
```

> By default the Jitsi client communicate via http to prosody.

Reviewer: ```suggestion
By default communication with Prosody is done using HTTP.
```

> If `ENABLE_XMPP_WEBSOCKET` is set to `1` the preferred communication method is WebSocket. http pooling as fallback is still

Reviewer: ```suggestion
If `ENABLE_XMPP_WEBSOCKET` is set to `1` the preferred communication method is WebSocket. HTTP is sstill available as a fallback.
```

Author: There was a typo
```suggestion
If `ENABLE_XMPP_WEBSOCKET` is set to `1` the preferred communication method is WebSocket. HTTP is still available as a fallback.
```

> available as fallback.

Reviewer: ```suggestion

```

Author: is this wrong?

> If Jitsi runs behind a reverse proxy make should the reverse proxy configuration supports WebSocket.

Reviewer: ```suggestion
If Jitsi runs behind a reverse proxy make sure the reverse proxy configuration supports WebSocket.
```
